plugins/kubernetes/app/assets/javascripts/kubernetes/services/kubernetes_service.js

```diff
@@ -0,0 +1,86 @@
+samson.service('kubernetesService', function($http, $q) {
+
+  var config = {
+    headers: {
+      'Accept': 'application/json'
+    }
+  };
+
+  this.loadKubernetesRoles = function(project_id) {
```

Comment:
You probably don't need to use 'Kubernetes' in each of the methods, as it belongs to a 'Kubernetes' service. It'll be obvious when using it from other code what it does.